.env.example

```diff
@@ -28,6 +28,14 @@ LOGGING_LEVEL=INFO
 #  Uncomment if using Windows for development.
 # COMPOSE_CONVERT_WINDOWS_PATHS=1
 
+<!--============================================================================
+= init variables                                                               =
+=============================================================================-->
+NEXT_ADMIN_NAME=Next Administrator
```

Comment:
Can you move the ENABLE_NEXT_BASE_USE variable to here?  So they are grouped appropriately?

Reply:
that's fair, I'll move it